### ISSUE TITLE
[Feature]: Add Privacy Safe Analytics 

### ISSUE THREAD

**Issue by johnhess** [Lookyloo/lookyloo#415]
### Is your feature request related to a problem? Please describe.

We'd [incl @Rafiot] like to understand the usage of the API vs. the UI, both in aggregate quantity and how additional fields are used at capture time.

### Describe the solution you'd like

Use [Clean Insights](cleaninsights.org) (along with Lookyloo's internal stats) to capture aggregate information about captures:

- Was a capture via API or UI?
- Which fields were set to which values (for public captures)?
- Which fields were changed from the default (for private captures)?

Implementation details:

- Clean Insights and the internal stats collection would be implemented server side with CI's Python SDK.  
- Especially for non-public captures, we would store either counters or record event with imprecise timestamps (i.e. dates, not times).
- No PII would be collected or stored.
- No capture cookies would be collected or stored.

### Describe alternatives you've considered

- Lookyloo internal stats alone: 
    - Workable.  
    - Using CI (or any dedicated analytics package) provides common analysis functions out of the box without needing to re implement them.  
    - Using CI+stats offers options not yet supported by stats alone, e.g. client-side analytics, aggregation of reports from multiple servers.
- Commercial analytics: 
    - 3P data sharing is not ideal for this project
- Generic self-hosted analytics (e.g. Matomo): 
    - Reasonable, but gathers data more broadly than necessary by default.  
    - There is some irony in a JS analytics package on Lookyloo, even if it's implemented in a privacy safe way.

### Additional context

Discussed this with @Rafiot this morning (edit/comment if I'm off base, please).

**Comment by Rafiot** [comment]
Yep, that's all good and in line with what we discussed! Thank you for the wrap up.

**Comment by johnhess** [comment]
@Rafiot is this still something you're interested in?
